service/routes.py

```diff
@@ -59,159 +59,200 @@ def index():
+inventory_args.add_argument('restock', type=int, location='args', required=False, help='List Inventory Items by Restock Level')
+inventory_args.add_argument('quantity', type=int, location='args', required=False, help='List Inventory Items by Quantity')
 
 ######################################################################
 #  R E S T   A P I   E N D P O I N T S
 ######################################################################
 
 
 ######################################################################
-# ADD A NEW INVENTORY ITEM
+#  PATH: /inventory/{inventory_id}
 ######################################################################
-@app.route("/inventory", methods=["POST"])
-def create_inventory_item():  # Replace entry with item
+@api.route('/inventory/<inventory_id>')
+@api.param('inventory_id', 'The Inventory identifier')
+class InventoryResource(Resource):
     """
-    Creates an inventory item
+    InventoryResource class
 
-    This endpoint will create an item based on the data in the body that is posted
+    Allows the manipulation of a single Inventory Item
+    GET /inventory/{inventory_id} - Returns an Item with the inventory_id
+    PUT /inventory/{inventory_id} - Update an Item with the inventory_id
+    DELETE /inventory/{inventory_id} -  Deletes an Item with the inventory_id
     """
-    app.logger.info("Request to create an inventory item")
-    check_content_type("application/json")
-    item = Inventory()
-    item.deserialize(request.get_json())
-    item.create()
-    message = item.serialize()
-    location_url = url_for("get_inventory", inventory_id=item.id, _external=True)
 
-    app.logger.info("Inventory item named [%s] with ID [%s] created.", item.name, item.id)
-    return jsonify(message), status.HTTP_201_CREATED, {"Location": location_url}
+    ######################################################################
+    # RETRIEVE AN INVENTORY ITEM
+    ######################################################################
+    @api.doc('get_inventory_items')
+    @api.response(404, 'Inventory Item not found')
+    @api.marshal_with(inventory_model)
+    def get(self, inventory_id):
+        """
+        Retrieve a single Inventory
+
+        This endpoint will return a Inventory based on it's id
+        """
+        app.logger.info("Request for item with id: %s", inventory_id)
+        inventory = Inventory.find(inventory_id)
+        if not inventory:
+            abort(status.HTTP_404_NOT_FOUND, f"Inventory with id '{inventory_id}' was not found.")
+
+        app.logger.info("Returning item: %s", inventory.name)
+        return inventory.serialize(), status.HTTP_200_OK
+
+    ######################################################################
+    #  UPDATE AN INVENTORY ITEM
+    ######################################################################
+
+    @api.doc('update_inventory_items')
+    @api.response(404, 'Pet not found')
```

Comment:
Inventory item not found

Reply:
I am aware of BDD test cases failing, it's getting difficult for me to check on my end since BDD test cases are failing for master too. I get a time out error

Reply:
You can reproduce the steps in the scenario on your browser and replace the message in the scenario with what's on your browser. Once you push your commit upstream, the BDD tests will un automatically.